File: mangerexample/src/main/java/com/manager/mangerexample/DTO/LigaDTO.java
package com.manager.mangerexample.DTO;

import javax.validation.constraints.NotBlank;

public class LigaDTO {
    @NotBlank
    private String nombreLiga;
    @NotBlank
    private String logo;
    @NotBlank
    private String direccion;

    public String getDireccion() {
        return direccion;
    }

    public void setDireccion(String direccion) {
        this.direccion = direccion;
    }

    public String getLogo() {
        return logo;
    }

    public void setLogo(String logo) {
        this.logo = logo;

    }

    public String getNombreLiga() {
        return nombreLiga;
    }

    public void setNombreLiga(String nombreLiga) {
        this.nombreLiga = nombreLiga;
    }

}

File: VYT/src/app/interfaces/liga-dto.ts
export class LigaDTO {
    nombreLiga:string;
    logo:string
    direccion:string
    constructor (nombreLiga:string,direccion:string,logoLiga:string){
        this.nombreLiga=nombreLiga;
        this.direccion=direccion;
        this.logo=logoLiga;
    }
}
